### ISSUE TITLE
Viewer component not showing

### ISSUE THREAD

**Issue by girish-io** [darwin-education/resium#418]
I followed the steps over at [craco-cesium](https://github.com/darwin-education/craco-cesium) to get an app created with `create-react-app` up and running but whenever I build the app using `npm run build` and serve the files using a webserver, you can quickly see the Cesium Viewer flash but quickly after it just disappears. The browser console (Chrome) shows no errors at all.

![browser-screenshot](https://i.imgur.com/JHgd1yH.gif)

_package.json_
```js
{
  "name": "cesium-test",
  "version": "0.1.0",
  "private": true,
  "dependencies": {
    "@craco/craco": "^5.6.4",
    "@testing-library/jest-dom": "^4.2.4",
    "@testing-library/react": "^9.5.0",
    "@testing-library/user-event": "^7.2.1",
    "cesium": "^1.70.1",
    "craco-cesium": "^1.2.0",
    "react": "^16.13.1",
    "react-dom": "^16.13.1",
    "react-scripts": "3.4.1",
    "resium": "^1.12.0-beta.1"
  },
  "scripts": {
    "start": "craco start",
    "build": "craco build",
    "test": "craco test",
    "eject": "react-scripts eject"
  },
  "eslintConfig": {
    "extends": "react-app"
  },
  "browserslist": {
    "production": [
      ">0.2%",
      "not dead",
      "not op_mini all"
    ],
    "development": [
      "last 1 chrome version",
      "last 1 firefox version",
      "last 1 safari version"
    ]
  }
}

```
_craco.config.js_

```js

module.exports = {
  plugins: [
    {
      plugin: require("craco-cesium")()
    }
  ]
};
```

_src/App.js_
```js
import React from 'react';
import { Viewer } from 'resium';

function App() {
  return (
    <Viewer />
  );
}

export default App;

```
_src/index.js_
```js
import React from 'react';
import ReactDOM from 'react-dom';
import App from './App';

ReactDOM.render(
  <React.StrictMode>
    <App />
  </React.StrictMode>,
  document.getElementById('root')
);

```

_public/index.html_
```html

<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8" />
    <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
    <meta name="viewport" content="width=device-width, initial-scale=1" />
    <meta name="theme-color" content="#000000" />
    <meta name="description" content="Web site created using create-react-app" />
    <link rel="apple-touch-icon" href="%PUBLIC_URL%/logo192.png" />
    <link rel="manifest" href="%PUBLIC_URL%/manifest.json" />
    <title>React App</title>
  </head>
  <body>
    <noscript>You need to enable JavaScript to run this app.</noscript>
    <div id="root"></div>
  </body>
</html>
```

Any help regarding this issue would be really appreciated.

**Comment by girish-io** [comment]
Downgrading the resium package from version `1.12.0-beta.1` to `1.11.0` seems to have fixed this issue. The viewer now shows up normally.

**Comment by rot1024** [comment]
Thank you for reporting. I have already recognize the bug and I am investigating a cause.

**Comment by rot1024** [comment]
The bug has been fixed and `v1.12.0-beta.2` has been released!
